ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -326,7 +326,6 @@ tests:
 - always_run: false
   as: e2e-aws-ovn-ipsec-serial
   optional: true
-  skip_if_only_changed: ^(?:docs|\.github)/|\.md$|^(?:\.gitignore|OWNERS|OWNERS_ALIASES|PROJECT|LICENSE)$
```

Comment:
curious as to what/why this is getting removed? is this what was making it run every time even though it was
'always_run: false' ?

if that's right, can you add that info to the commit message?